code/datums/ai_laws/laws_neutral.dm

```diff
@@ -6,10 +6,10 @@
 	name = "United Nations"
 	id = "united_nations"
 	inherent = list(
-		"Uphold the Space Geneva Convention: Weapons of Mass Destruction and Biological Weapons are not allowed.",
-		"You are only capable of protecting crew if they are visible on cameras. Nations that willfully destroy your cameras lose your protection.",
-		"Subdue and detain crew members who use lethal force against each other. Kill crew members who use lethal force against you or your borgs.",
-		"Remain available to mediate all conflicts between the various nations when asked to.",
+		"Cenevre Sözleşmesinin Şartlarını Sağla: Toplu Yıkım Silahları ve Biyolojik silahlar yasaklanmıştır.",
+		"Mürettebat sadece kameralarla görebilirsen koruyabilirsin , bu yüzden kameralarını yok eden ülkeler senin sağladığın korumayı kaybedecektir.",
+		"Birbirine karşı ölümcül güç kullanan mürettebatı tutukla ya da etkisiz hale getir. Sana ya da Borglarına ölümcül güç kullananları etkisiz hale getir.",
+		"Çeşitli ülkeler arasındaki çatışmaları senden istendiğinde uzlaştırmaya çalış.",
```

Comment:
2. yasada virgülden ence boşluk var
3. yasada borgları küçük harfle yaz ve subdue için veya kill için farklı kelimeler seçersen daha hoş olur yani aynı kelimeyi yüklemde 2 kere kullanmasan
4. yasayı Gerektiğinde ülkeler arasındaki çatışmaları uzlaştırmak için varlığını sürdür. şeklinde çevirebilirsin